bHY6yh-de1103/cod-vyg67d_bHY6yh-de1103.xml

```diff
@@ -0,0 +1,168 @@
+            <author ref="#per-c7y65a">Geremia da Montagnone</author>
+            <respStmt>
+               <name ref="#jeffreycwitt">Jeffrey C. Witt</name>
+               <resp>Transcription Editor</resp>
+               <resp>TEI Encoder</resp>
+            </respStmt>
+         </titleStmt>
+         <editionStmt>
+            <edition n="0.0.0-dev">
+               <title>Rubrica 5</title>
+               <date when="2023-06-07">June 07, 2023</date>
+            </edition>
+         </editionStmt>
+         <publicationStmt>
+            <authority>SCTA</authority>
+            <availability status="free">
+               <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+               <witness xml:id="O" n="cod-vyg67d">Vatican, Biblioteca Apostolica Vaticana, Vat. lat. 8175</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-06-07" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#O" n="10-r"/>
+            <cb ed="#O" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-de1103">
+            <head xml:id="bHY6yh-de1103-Hd1e108">Rubrica 5</head>
+            <head xml:id="bHY6yh-de1103-Hd1e111" type="question-title">De retributione Dei</head>
+            <p xml:id="bHY6yh-de1103-d1e114">
+                  <lb ed="#O"> ¶ De retributione dei Rubrica. 5. 
+    </p>
+            <p xml:id="bHY6yh-de1103-d1e119">
+                  <lb ed="#O"><ref xml:id="bHY6yh-de1103-Rd1e116">Iob hystoria 34. c. </ref> 
+                    <quote xml:id="bHY6yh-de1103-Qd1e121" source="http://scta.info/resource/iob34_11@1-10">¶ Opus hominis 
+                  <lb ed="#O">reddet ei deus: et iuxta via 
+                  <lb ed="#O">singulorum restituet. </quote>
+            </p>
+            <p xml:id="bHY6yh-de1103-d1e131">
+              <ref xml:id="bHY6yh-de1103-Rd1e132">In genesi. c. 4. </ref> <quote xml:id="bHY6yh-de1103-Qd1e135" source="http://scta.info/resource/gen4_7@1-5">
+                  <lb ed="#O"> Nonne si bene egeris recipies. </quote> 
```

Comment:
Be careful not to have too many spaces in the text. For example you have an extra space before your /ref and a space before and after your quote.